components/atoms/SnackbarNotification.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+class SnackbarNotification extends Component {
+    static propTypes = {
+        /** Unique key for identification */
+        id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+        /** Message or content of the notification */
+        text: PropTypes.string,
+        /** Context of notification (success, info, warning, danger, hidden) */
+        context: PropTypes.string,
+        /** Context of close button, if not passed uses context value (succes, info, warning, danger, dark) */
+        buttonContext: PropTypes.string,
+        /** Time in ms to hide notification, if 0 is passed ignores timeout */
+        msToClose: PropTypes.number,
+        /** Hides the close button, if true auto enables timeout to hide notification */
+        hideCloseButton: PropTypes.bool,
+        /** Function to be called after hiding notification */
+        callback: PropTypes.func,
+        /** Component children that can be used as content */
+        children: PropTypes.node,
+        /** Styles passed through props */
+        style: PropTypes.object,
+        /** Class names passed through props */
+        className: PropTypes.string
+    };
+
+    static defaultProps = {
+        msToClose: 3000,
+        hideCloseButton: false,
+        style: {},
```

Comment:
I would prefer to just remove the `style` and `className` defaults, having `undefined` shouldn't break anything and makes snapshots cleaner

Reply:
Thanks, fixed!